src/PgManyToManyRelationPlugin.js

```diff
@@ -52,23 +474,147 @@ module.exports = function PgManyToManyRelationPlugin(
             .join("-and-")}-list`
         );
       },
+      edgeViaJunction(nodeTypeName, junctionTypeName) {
```

Comment:
How do you feel about the word "via" compared to the word "by"? Should this be named `edgeByJunction` instead? When I was writing this code initially, I used the word "via" to make it clearer which functions I wrote (compared to which were already there), but now that there's a good chance this feature might get merged, I think it's more consistent to use "by" instead.

Reply:
No preference either way; I'll let you make the call.